src/en/models-config.md

```diff
@@ -1,72 +1,135 @@
 Title: General configuration options  
-Commands: get-env, set-env  
+TODO: check options haven't changed
 
-# General configuration options
+# Configuring models
 
 There may be times, particularly when dealing with complex cloud environments
 or running multiple instances of Juju, where it would be useful to control
-further aspects of the Juju environment. Many options can be changed by adding
-extra key:value pairs to the environments.yaml file which Juju uses to initiate
-environments. Options which are specific to certain cloud providers are
-detailed in the individual configuration pages, but there are additional
-settings which can be applied to any environment. Some of these may be useful
-for particular use cases as outlined in the sections below, or you can
-[skip to the table](#alphabetical-list-of-general-configuration-values) of all
-the values.
+further aspects of the models you create. 
```

Comment:
may want to state explicitly that any machines created in the configured model will inherit these settings